datastore/importers/nmdb-import-cisco-show-vrf/nmdb-import-cisco-show-vrf.cpp

```diff
@@ -0,0 +1,96 @@
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+// =============================================================================
+// Maintained by Sandia National Laboratories <Netmeld@sandia.gov>
+// =============================================================================
+
+#include <netmeld/datastore/tools/AbstractImportSpiritTool.hpp>
+#include <netmeld/datastore/parsers/ParserHelper.hpp>
+
+#include "Parser.hpp"
+
+namespace nmdo = netmeld::datastore::objects;
+namespace nmdp = netmeld::datastore::parsers;
+namespace nmdt = netmeld::datastore::tools;
+
+
+// =============================================================================
+// Import tool definition
+// =============================================================================
+template<typename P, typename R>
+class Tool : public nmdt::AbstractImportSpiritTool<P,R>
+{
+  // ===========================================================================
+  // Variables
+  // ===========================================================================
+  private:
+  protected:
+  public:
+
+
+  // ===========================================================================
+  // Constructors
+  // ===========================================================================
+  private: // Constructors should rarely appear at this scope
+  protected: // Constructors intended for internal/subclass API
+  public: // Constructors should generally be public
+    Tool() : nmdt::AbstractImportSpiritTool<P,R>
+      (
+       "show vrf [detail]",
+       PROGRAM_NAME,
+       PROGRAM_VERSION
+      )
+    {}
+
+
+  // ===========================================================================
+  // Methods
+  // ===========================================================================
+  private:
+    void
+    specificInserts(pqxx::transaction_base& t) override
+    {
+      const auto& toolRunId {this->getToolRunId()};
+      const auto& deviceId  {this->getDeviceId()};
+
+      for (auto& results : this->tResults) {
+        LOG_DEBUG << "Iterating over Vrfs\n";
+        for (auto& result : results.vrfs) {
+          result.save(t, toolRunId, deviceId);
+          LOG_DEBUG << result.toDebugString() << std::endl;
```

Comment:
Swap the order of these two.
```suggestion
          LOG_DEBUG << result.toDebugString() << std::endl;
          result.save(t, toolRunId, deviceId);
```